### PR TITLE
Change aiohttp services (db and redis) startup

### PR DESCRIPTION
**Pull request by CatCanCreate**

### CODE REVIEW

File: create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/app.py [hunk 1]
```diff
@@ -33,19 +33,22 @@ def init_jinja2(app: web.Application) -> None:
 {%- if cookiecutter.use_postgres == 'y' %}
 
 
-async def init_database(app: web.Application) -> None:
+async def database(app: web.Application) -> None:
     '''
     This is signal for success creating connection with database
```
Comment (Arfey) [inline create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/app.py]: U need update docstring. Now it's not only `startup` function 😀

Comment (CatCanCreate) [reply create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/app.py]: @Arfey done

**Comment by Arfey** [comment]
thx 👍

**Comment by hzlmn** [comment]
Would be nice to pin aiohttp version if so, this change won't work on 3.0 and below.

**Comment by Arfey** [comment]
@hzlmn good point. I created issue for that https://github.com/aio-libs/create-aio-app/issues/49

**Pull request merged**